addons/html_editor/static/src/core/selection_plugin.js

```diff
@@ -174,11 +174,11 @@ export class SelectionPlugin extends Plugin {
         "extractContent",
         "preserveSelection",
         "resetSelection",
-        "getSelectedNodes",
-        "getTraversedNodes",
-        "getTraversedBlocks",
```

Comment:
We need to keep those to comply with the stable policy.

Reply:
Where should they lead to then?... These functions don't exist anymore.